udaykalra/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -40,11 +61,20 @@
     availableRanges.add(fullDay);
 
     // Obtain data from the request.
-    Collection<String> attendees = request.getAttendees();
+    Collection<String> attendees = new HashSet<>();
+    Collection<String> mandatoryAttendees = request.getAttendees();
+    Collection<String> optionalAttendees = request.getOptionalAttendees();
     long requestDuration = request.getDuration();
 
     Collection<TimeRange> busyRanges = new HashSet<>();
 
+    attendees.addAll(mandatoryAttendees);
```

Comment:
Could you initialize attendees with mandatoryAttendees, instead of adding it later? That would avoid the need for the extra mandatoryAttendees var.

Reply:
Done